content/participants/ttjc@SagarPai99/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Sagar Sudesh Pai
+type: "participant"
+name: "Sagar Sudesh Pai"
+portfolio: "https://tanaypratap.com"
+github-link: "https://github.com/SagarPai99"
+linkedin-link: "https://www.linkedin.com/in/sagar-sudesh-pai/"
+looking-for: "job/internship"
+description: "React | Javascript | Node | Bootstrap | SQL | MongoDB | Machine Learning"
+---
+
+Hi! I am Sagar. I am currently an ongoing undergrad with a passionate interest towards web development.
+
+### introduction
+
+I am an enthusiastic learner and love to learn new stuff. 
+
+## why are you taking this challenge?
+
+I need a job in December 2020.
+I want to learn about new things and get to know people to gain insights from them.
+
+## something interesting about you?
+
+I am an avid food lover !
+
+### help I need?
+
+It would be good if someone guides me the path towards building a career in web dev.
+
+### how to contact me?
+
+- [Twitter](https://twitter.com/SagarPai99)
+- [LinkedIn](https://www.linkedin.com/in/sagar-sudesh-pai/)
+
+### projects
+
+not a lot of projects are done. I will keep updating it with real links when done.
+
+My projects:
```

Comment:
Remove the default project (Binder) and blog (why i like GraphQL over REST). You can add your own project and blogs here ones you complete them.